views/layouts/main.hbs

```diff
@@ -51,6 +45,15 @@
 	<main>
 		{{{ body }}}
 	</main>
+
+	<footer class="page-footer">
+		<div class="footer-copyright">
+			<div class="container">
+				© 2021 CollabWhiteboard
+			</div>
+		</div>
```

Comment:
I am not stopping this pr for this but the indents are a bit large.

Reply:
Very weird... definitely looks like unnecessary indentation here, but looks normal in VS code.